mcl_3dl:
  map_frame: "map"
  odom_frame: "base_link"
  accum_cloud: 1
  lpf_step: 0.0
  num_particles: 256
  resample_var_x: 0.2
  resample_var_y: 0.2
  resample_var_z: 0.05
  bias_var_dist: 100.0
obj_to_pointcloud:
  frame_id: "map"
  points_per_meter_sq: 1200.0
  downsample_grid: 0.1
pcd_to_pointcloud:
  frame_id: "map"
  latch: true
track_odometry:
  z_filter: 1.0
  tf_tolerance: 0.1
  without_odom: true

